feature/#393 added doc for return type of submit methods (#396)
Avaiga/taipy-doc: docs/manuals/core/entities/scheduling-and-job-execution.md

In this section, it is assumed that <a href="./code_example/my_config.py" download>`my_config.py`</a>
module contains a Taipy configuration already implemented.

> To execute a scenario, you need to call the `taipy.submit()^` method, which will return a list of created `Job^`:

Reviewer: ```suggestion
To execute a scenario, you need to call the `taipy.submit()^` method. It returns a list of created `Job^`s:
```

taipy waits until all the submitted jobs are finished, or the timeout expires (which ever occurred
first).

It is also possible to submit a single pipeline using the same `taipy.submit()^` method, which will

> return a list of created `Job^`:

Reviewer: ```suggestion
It is also possible to submit a single pipeline using the same `taipy.submit()^` method. . It returns a list of created `Job^`s:
```

    pipeline.submit()
    ```

> You can also submit a single task with the same `taipy.submit()^` method, and it will return the created `Job^`:

Reviewer: ```suggestion
You can also submit a single task with the same `taipy.submit()^` method. It returns the created `Job^`:
```
